.ci/scripts/setup-macos.sh

```diff
@@ -28,15 +28,8 @@ install_buck() {
   fi
 
   pushd .ci/docker
-
-  # TODO(huydo): This is a one-off copy of buck2 2024-05-15 to unblock Jon and
```

Comment:
Please keep a comment here explaining why this copy exists instead of getting in the way we get it for non-macos systems. I'd still consider it a TODO to remove this manually-managed copy.

Reply:
It's also important to let people know who to talk to when we need to upgrade the buck2 version and add a new copy to the S3 bucket